utils/find_replace.py

```diff
@@ -0,0 +1,129 @@
+"""
+    Python script to upload and replace broken links with new ones.
+
+    Example Usage: python utils/find_replace.py --csv input.csv --out output.csv
+
+    Outputs a csv with meeting_id previous_url previous_filesystem_path new_file_url file_id event_file_id columns
+"""
+import argparse
+import csv
+import logging
+import os
+import shutil
+import tempfile
+from collections import namedtuple
+
+from pmg import app, db
+from pmg.models.resources import CommitteeMeeting, EventFile, File
+
+
+def create_logger():
+    logging.basicConfig(level=logging.INFO, filemode="a", encoding="utf-8")
+    formatter = logging.Formatter("%(asctime)s | %(levelname)s | %(message)s")
+    logger = logging.getLogger(__name__)
+    file_handler = logging.FileHandler(args.logfile)
+    file_handler.setFormatter(formatter)
+    logger.addHandler(file_handler)
+    return logger
+
+
+def read_csv(path):
+    with open(path, "r") as f:
+        reader = csv.DictReader(f)
+        for row in reader:
+            if row["possible_duplicate"] == "True":
+                continue
+            yield row
+
+
+def find_meeting_by_id(id):
+    meeting = db.session.query(CommitteeMeeting).filter_by(id=int(id)).first()
+    return meeting
+
+
+def upload_file(path):
+    with tempfile.TemporaryDirectory() as temp_dir:
+        temp_path = os.path.join(temp_dir, os.path.basename(path))
+        shutil.copyfile(path, temp_path)
+        file = File()
+        file.from_file_blob(temp_path)
+        file.file_path = path
```

Comment:
should we be overwriting this property? does it not get sit by from_file_blob?

Reply:
No, we shouldn't. Removed.